governance.md

```diff
@@ -105,7 +99,7 @@ The secondary decision making process is done by voting. The following items mus
 * Licensing and intellectual property changes such as new logos or wordmarks (simple majority)
 * Adding, archiving, or removing projects (simple majority)
 
-Other decisions may, but do not need to be, called out and put up for decision  on the InstructLab community Slack channel. Directions to join the Slack channel can be found [here](https://github.com/instructlab/community/blob/main/InstructLabSlackGuide.md). In the future, a mailing list will be established. This can be done by anyone at any time. By default, any decisions called to a vote will be for a _simple majority_ vote.
+Other decisions may, but do not need to be, called out and put up for decision on the InstructLab community Slack channel. Directions to join the Slack channel can be found [here](https://github.com/instructlab/community/blob/main/InstructLabSlackGuide.md). In the future, a mailing list will be established. This can be done by anyone at any time. By default, any decisions called to a vote will be for a _simple majority_ vote.
```

Comment:
Mailings list have been established. This would be discussed on the community list.

https://github.com/instructlab/community/blob/main/Collaboration.md#email-lists